plugin/wavesurfer.regions.js

```diff
@@ -89,6 +89,9 @@ WaveSurfer.Region = {
         this.color = params.color || 'rgba(0, 0, 0, 0.1)';
         this.data = params.data || {};
 
+        this.maxWidth = params.maxWidth === undefined ? null : Number(params.maxWidth);
+        this.minWidth = params.minWidth === undefined ? null : Number(params.minWidth);
```

Comment:
* Rename `*Width` to `*Length`.
 * Just `this.*Length = params.*Length` would be enough.